From b40c432bdc305f6dedc828a607d06f19ea44188f Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.liu@freescale.com>
Date: Thu, 27 Sep 2012 15:36:04 +0800
Subject: [PATCH 070/310] ENGR00223797 MX6 SabreSD:Add fb0 base addr to kernel
 cmdline

This patch adds 'fb0base=0x27b00000' to kernel command line
for MX6 SabreSD platform to specify fb0 base address to be
0x27b00000, so that kernel can use a reserved fb memory
starting from this address. This address is used in uboot
fb. To support smooth UI transition from uboot to kernel,
this fb memory has to be reserved by kernel, otherwise,
it is likely to be overwritten by others, so kernel will
go on using this block of memory to save memory usage.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
---
 sabresd_6dq/BoardConfig.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sabresd_6dq/BoardConfig.mk b/sabresd_6dq/BoardConfig.mk
index 266fea7..68a4807 100755
--- a/sabresd_6dq/BoardConfig.mk
+++ b/sabresd_6dq/BoardConfig.mk
@@ -6,12 +6,12 @@ include device/fsl/imx6/soc/imx6dq.mk
 include device/fsl/imx6/sabresd/SabreSDBoardConfigComm.mk
 include device/fsl-proprietary/gpu-viv/fsl-gpu.mk
 
-BOARD_KERNEL_CMDLINE := console=ttymxc0,115200 init=/init video=mxcfb0 video=mxcfb1:off video=mxcfb2:off fbmem=10M vmalloc=400M androidboot.console=ttymxc0 androidboot.hardware=freescale
+BOARD_KERNEL_CMDLINE := console=ttymxc0,115200 init=/init video=mxcfb0 video=mxcfb1:off video=mxcfb2:off fbmem=10M fb0base=0x27b00000 vmalloc=400M androidboot.console=ttymxc0 androidboot.hardware=freescale
 
 
 ifeq ($(TARGET_USERIMAGES_USE_UBIFS),true)
 #UBI boot command line.
-BOARD_KERNEL_CMDLINE := console=ttymxc0,115200 init=/init video=mxcfb0 video=mxcfb1:off video=mxcfb2:off fbmem=10M vmalloc=400M androidboot.console=ttymxc0  mtdparts=gpmi-nand:20m(bootloader),20m(bootimg),20m(recovery),-(root) gpmi_debug_init ubi.mtd=3 androidboot.hardware=freescale 
+BOARD_KERNEL_CMDLINE := console=ttymxc0,115200 init=/init video=mxcfb0 video=mxcfb1:off video=mxcfb2:off fbmem=10M fb0base=0x27b00000 vmalloc=400M androidboot.console=ttymxc0  mtdparts=gpmi-nand:20m(bootloader),20m(bootimg),20m(recovery),-(root) gpmi_debug_init ubi.mtd=3 androidboot.hardware=freescale
 endif
 
 
-- 
1.8.0

